4001-5000/LIT4859Litany.xml

```diff
@@ -63,7 +63,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <citedRange>10</citedRange>
                </bibl>
             </listBibl>
-            </div>
+
+            <div type="textpart" xml:lang="gez">
+    <ab>
```

Comment:
Could you indicate from where this text is taken?

Reply:
I took the information on "ehud" from my the edition of the MQ that I have at home, I will include editions in the record as soon as I reach them in the office. I could not reach now to bm:MashafaSalotaKidan1960, the only source where it is allegedly under "sellus", but I am nearly sure that it is not like this.

Reply:
Checked.